tests/acceptance/test_dashboard.py

```diff
@@ -74,3 +74,10 @@ def test_new_dashboard(self):
             self.browser.get(self.path)
             self.browser.wait_until_not('.loading-indicator')
             self.browser.snapshot('new dashboard')
+
+    def test_new_dashboard_empty(self):
+        with self.feature('organizations:dashboard'):
```

Comment:
This snapshot isn't working, you may need to just create a new user/org

Reply:
👍